src/docs/cookbook/networking/update-data.md

```diff
@@ -9,39 +10,41 @@ next:
 ---
 
 Updating data over the internet is necessary for most apps.
-The `http` package has got it covered too.
+The `http` package has got that covered!
 
 This recipe uses the following steps:
 
-  1. Add the `http` package.
-  2. Update data over the internet using the `http` package.
-  3. Convert the response into a custom Dart object.
-  4. Get data from internet
-  5. Update existing `title` from user input
-  6. Update and display the response on screen.
+  1. Add the `http` package
+  2. Update data over the internet using the `http` package
+  3. Convert the response into a custom Dart object
+  4. Get the data from the internet
+  5. Update the existing `title` from user input
+  6. Update and display the response on screen
```

Comment:
See above re: punctuation in numbered lists of steps.
https://developers.google.com/style/lists#capitalization-and-end-punctuation

![Screen Shot 2020-03-10 at 4 52 51 PM](https://user-images.githubusercontent.com/5550175/76369094-9d100500-62ef-11ea-957a-3a57c5f158b5.png)